templates/expenses/show.html

```diff
@@ -228,6 +228,8 @@ <h3>Eller skicka en färdig kommentar:</h3>
                 <ul>
                     <li><a href="#" v-on:click="send">Jag attesterar också detta.</a></li>
                     <li><a href="#" v-on:click="send">Det här har jag inte godkänt.</a></li>
+                    <li><a href="#" v-on:click="send">Det här överskrider budgeten.</a></li>
+                    <li><a href="#" v-on:click="send">Ett kvitto per utlägg.</a></li>
```

Comment:
```suggestion
                    <li><a href="#" v-on:click="send">Varje kvitto ska laddas upp som ett eget utlägg på Cashflow.</a></li>
```
Maybe this is a bit clearer?